index.html

```diff
@@ -6,13 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Please deploy the portfolio site to GitHub pages or Netlify and add the live link to README.